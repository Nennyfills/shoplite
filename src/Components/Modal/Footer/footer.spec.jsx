import React from 'react';
import { shallow } from 'enzyme';

import { checkProps } from 'Test/propsTypes';
import ModalFooter from '.';

const props = {
  total: 0,
  currencies: 'NGN',
};

const setUp = (props = {}) => {
  const component = shallow(<ModalFooter {...props} />);
  return component;
};

describe('Modal Footer', () => {
  let component;
  beforeEach(() => {
    component = setUp(props);
  });
  it('Should render without errors', () => {
    expect(component).toMatchSnapshot();
  });
  it('Should render Body div', () => {
    const wrapper = component.find(`[data-test='modal-footer']`);
    expect(wrapper.length).toBe(1);
  });
  it('Checking PropTypes should not throw error', () => {
    const propErr = checkProps(ModalFooter, props);
    expect(propErr).toBeUndefined();
  });
});
