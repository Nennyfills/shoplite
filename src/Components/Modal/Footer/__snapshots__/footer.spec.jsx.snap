// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Footer Should render without errors 1`] = `ShallowWrapper {}`;
